libraries/lib-wave-track/PasteOverPreservingClips.cpp

```diff
@@ -50,9 +50,10 @@ ClipData CollectClipData(
    return results;
 }
 
-void PasteOverPreservingClips(const ClipData &data,
-   WaveTrack &oldTrack, sampleCount start, sampleCount len,
-   const WaveTrack &newContents)
+void PasteOverPreservingClips(
+   const ClipData& data, WaveTrack& oldTrack, sampleCount start,
+   sampleCount len, const WaveTrack& newContents,
+   std::function<void(double)> reportProgress)
```

Comment:
I mentioned in another review today that there is remapping of split lines in some of the functions but there is not-exactly-duplicated code doing it.

If that were unified, this would likely become the common utility.  But if only effects ever use this, they would always pass a no-op progress reporter anyway, for the same reasons as commented in EQ, right?

So maybe this function needs no new argument and just supplies the no-op to `Join` always.